gulpfile.js

```diff
@@ -60,3 +67,23 @@ gulp.task(
     }),
   ),
 );
+
+gulp.task(
+  'test:wtr',
+  gulp.parallel('build-test-css', () => {
+    const outputDir = 'build/scripts';
+    const testFiles = [
+      'test/bootstrap.js',
+      ...globSync('src/**/*-test.js').sort(),
+    ];
+
+    const testSource = testFiles
+      .map(path => `import "../../${path}";`)
+      .join('\n');
+
+    mkdirSync(outputDir, { recursive: true });
+    writeFileSync(`${outputDir}/test-inputs.js`, testSource);
+
+    return buildJS('./rollup-tests.config.js');
```

Comment:
This is more or less copied from frontend-build's `runTests`, but excluding the part about starting a karma server and removing all dynamic logic.